web-extension/scripts/load-bookmarks.js

```diff
@@ -79,28 +78,15 @@ BmcMangaList.prototype.onSourceClick = function(comic, source) {
         resource: {
             origin: window.location.origin,
             reader: source.reader,
+            comicId: comic.id,
             comic: Object.assign({
                 common: {
                     name: source.name,
```

Comment:
As per our offline discussion, you can retrieve this through:
`comic.getSource(source.reader).info.name` (assuming that `comic` is a `BmcComic` instance)